source/guide/3.2/plugin-softlayer.md

```diff
@@ -13,6 +13,20 @@ This section describes how to use a SoftLayer based cloud infrastructure in your
 For more information about SoftLayer, please refer to: [http://www.softlayer.com/](http://www.softlayer.com/).
 {%endsummary%}
 
+# Requirements
+  * A SoftLayer account, see [SoftLayer Authentication](#softlayer-authentication).
+  * [The commercial version of Cloudify](/goPro.html) 
+
+# Compatibility
+  The SoftLayer plugin uses the [SoftLayer for python 3.3.0](https://pypi.python.org/pypi/SoftLayer/3.3.0).
```

Comment:
i think there's a word missing somewhere in there
or maybe im just reading it wrong :)